### ISSUE TITLE
BUG: Templates Behave Inconsistently When Modified

### ISSUE THREAD

**Issue by SooBahkDo** [WPDevelopers/notificationx#67]
I used the advanced button to modify a template and published while the advanced section was open and the notifications will not display the Title even though that placeholder is still present.

If I close the advanced section before publishing, then all the modifications are ignored and the title does display in the notification.  (The modifications appear to be saved because they are still present the next time I open the advanced section)

**Comment by priyomukul** [comment]
can you give some more details, for example what you have setup for a custom advanced template. 

https://d.pr/free/i/YNp2Lo 

Currently we have this much options for Advanced Template. 

In my primary inspection, it works fine.

Our support team testing this again... i hope, we will fix it if anything goes wrong.

**Comment by SooBahkDo** [comment]
I discovered that I had reduced the font size for the 3rd row of text ({time} sometime ago) to 1px to make it not visible at all, but on the template I selected the font size setting actually changed the font size for the 2nd row of text, NOT the 3rd row.  That made me think the 2nd row {Title} was not displaying but it was.

Refer to screenshot.
![template-font-size-config-issue-2019-09-08_1638](https://user-images.githubusercontent.com/3432950/64496328-76521280-d269-11e9-8bda-9de28a3dd559.png)
